modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -35,6 +35,21 @@ The following premium plugin updates were released alongside {productname} 6.5.
 
 The following premium plugin has been announced as reaching its end-of-life:
 
+=== Format Painter 2.0.2
+
+The {productname} 6.5 release includes an accompanying release of the **Format Painter** premium plugin.
+
+**Format Painter** 2.0.2 includes one fix:
+
+=== It was possible to modify `contenteditable = "false"` text elements using formatpainter
+//#TINY-9472
+
+In previous versions of {productname}, the format painter tool previously allowed users to modify 'contenteditable = "false"' text elements.
+
+As a consequence, the format painter would apply formatting changes to the 'contenteditable = "false"' content.
+
+In {productname} 6.5, the format painter no longer affects 'contenteditable = "false"' text elements and the original formatting of the 'contenteditable = "false"' content remains intact.
+
```

Comment:
```suggestion
=== Format Painter 2.0.2

The {productname} 6.5 release includes an accompanying release of the **Format Painter** Premium plugin.

**Format Painter** 2.0.2 includes one fix:

=== It was possible to modify text elements with the `contenteditable="false"` attribute set using formatpainter
//#TINY-9472

Users could use previous versions of the Format Painter plugin to modify text elements with the `contenteditable="false"` attribute set.

That is, users could, contrary to the attribute setting, make formatting changes to text marked as Read Only using this plugin.

With this update, the Format Painter plugin marks the `'contenteditable="false"` attribute setting properly: text elements with this setting are no longer changed if this plugin is applied to them.

For information on the **Format Painter** plugin, see: xref:formatpainter.adoc[Format Painter].
```

1. Edits to focus this section on the plugin having been fixed. (This fix isn’t a change to TinyMCE, it’s a change to the plugin.)
2. Added the standard _For information_ line, with an xref pointing to the Format Painter documentation.

Reply:
> @brian-forte
> 
> 
> 
> With this update, the Format Painter plugin marks the `'contenteditable="false"`
> 
> _Update to_:
> 
> With this update, the Format Painter plugin marks the `contenteditable="false"`

Typo corrected. Thanks.